src/constants/games.ts

```diff
@@ -0,0 +1,12 @@
+import { GameKey, GameName } from '@/types/games';
+
+export const GAMES = {
+  'LEAGUE OF LEGENDS': '리그오브레전드',
+  BATTLEGROUNDS: '배틀그라운드',
+  'OVERWATCH 2': '오버워치 2',
+  MINECRAFT: '마인크래프트',
+} as const;
+
+export const GAME_KEY_LIST: GameKey[] = ['LEAGUE OF LEGENDS', 'BATTLEGROUNDS', 'OVERWATCH 2', 'MINECRAFT'];
```

Comment:
```suggestion
export const GAME_KEY_LIST: GameKey[] = Object.keys(GAMES) as GameKey[];
```
GAME_KEY_LIST에서 직접 문자열을 넣지 않고, 위처럼 GAMES에서 key값을 추출해서 배열로 만들어도 될 것 같습니다

Reply:
사실 원래 그렇게 하려고 했다가, Object 는 순서가 보장되지 않는다는걸 깨닫고 배열을 직접 적어준 거였습니다ㅠ
헤더에 표시될 게임명이니까 순서가 섞이면 안될거같아서요..!

Reply:
아 그래서 멘토님이 이 방법도 설명해주신 거였군요..? 객체 순서 유지를 위해서?!

![image](https://github.com/sprint-team3/GGF/assets/43297823/2aaaa3f4-b53f-4a6d-b804-aeb5d2108b46)

Reply:
아하!!

Reply:
위 코드로 수정했습니다!